packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -55,6 +56,9 @@ const Duration _kCursorBlinkWaitForStart = Duration(milliseconds: 150);
 // is shown in an obscured text field.
 const int _kObscureShowLatestCharCursorTicks = 3;
 
+// The minimum width of an iPad screen.
```

Comment:
Could you add to the comment about where you got this value and what iPad models it's for?  I'm thinking it would be helpful in case a new iPad comes out that would require this constant to be changed.